### PR TITLE
update of aqm_cap for coordinate change

### LINKED ISSUE #3: PM2.5 peaks are offset due to wrong coordinates
There needs to be a change from radians to degrees to match what UFS uses to fix an offset in the time of the PM2.5 peaks.

### PR DESCRIPTION
**Pull request by JianpingHuang-NOAA**
Update aqm_cap.F90 to fix the coordinate from radian to degree 

Fixes #3

### CODE REVIEW

File: src/aqm_cap.F90 [hunk 1]
```diff
@@ -60,6 +60,9 @@ module AQM
     /)
 
   private
+
+  real(8), parameter :: pi = 3.1415926535897931
+  real(8), parameter :: rad2deg = 180./pi
```
Comment (DusanJovic-NOAA) [inline src/aqm_cap.F90]: These parameters are used to modify `coord` array which has `ESMF_KIND_R8` kind, so I would declare these two parameters with the same kind value. Besides literal `8` in `real(8)` is not portable.

File: src/aqm_cap.F90 [hunk 2]
```diff
@@ -310,6 +318,10 @@ subroutine DataInitialize(model, rc)
     ! -- assume all fields on same grid and use first field
     call ESMF_StateGet(importState, field=field, &
       itemName=trim(importFieldNames(1)), rc=rc)
+!         write(msgString,*) &
+!          "hjp501a=", trim(importFieldNames(1)) 
+!          call ESMF_LogWrite(trim(name)//": hjp501:  &
+!          "//trim(msgString),ESMF_LOGMSG_INFO, rc=rc)
```
Comment (DusanJovic-NOAA) [inline src/aqm_cap.F90]: Remove commented debug prints.

File: src/aqm_cap.F90 [hunk 3]
```diff
@@ -413,14 +431,28 @@ subroutine DataInitialize(model, rc)
...
           if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, &
             line=__LINE__, &
             file=__FILE__)) &
             return  ! bail out
+          if (aqmGridCoordSys == ESMF_COORDSYS_SPH_RAD) then
+            coord = coord * rad2deg
+          endif
+           print *,'in AQM, get lon(0,0)=',coord(tlb(1),tlb(2)), 'tlb=',tlb,'tub=',tub &
+               ,'localPet=',localPet
+
```
Comment (DusanJovic-NOAA) [inline src/aqm_cap.F90]: Are these debug prints still necessary?

**Comment by junwang-noaa** [comment]
@JianpingHuang-NOAA @BrianCurtis-NOAA Would you please address Dusan's comments?

File: src/aqm_cap.F90 [hunk 4]
```diff
@@ -261,7 +265,11 @@ subroutine DataInitialize(model, rc)
     real(ESMF_KIND_R8)         :: dts
     type(ESMF_Time)            :: startTime
     type(ESMF_TimeInterval)    :: TimeStep
+    type(ESMF_CoordSys_Flag)   :: aqmGridCoordSys
     character(len=ESMF_MAXSTR) :: msgString, name
+!test:
+    integer :: tlb(2), tub(2)
+
```
Comment (DusanJovic-NOAA) [inline src/aqm_cap.F90:272]: now that the debug prints are removed, these are not necessary anymore.

Comment (BrianCurtis-NOAA) [reply src/aqm_cap.F90:272]: I think we can keep it for now. I will discuss with Jianping and we can adjust this in the next PR for AQM.

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Pull request merged**